### ISSUE TITLE
TypeError: Cannot read property 'getAttribute' of null

### ISSUE THREAD

**Issue by kiriruru** [AlexxNB/tinro#12]
Hi!

After update tinro to 0.2.7, if click on any element, excluding routed links, I see error in console:
```
Uncaught TypeError: Cannot read property 'getAttribute' of null
    at tinro_lib.js:1
    at Array.reduce (<anonymous>)
    at i$1 (tinro_lib.js:1)
    at e (tinro_lib.js:1)
(anonymous) @ tinro_lib.js:1
i$1 @ tinro_lib.js:1
e @ tinro_lib.js:1
```

**Comment by AlexxNB** [comment]
Fixed in v0.2.8. Thanks!

**Comment by kiriruru** [comment]
Very quickly! Thank you!

Best regards,
Kirill Boldyrev
--
https://sitefactor.ru/
Phone/telegram: +7 910 469 08 66
WhatsApp +371 278 50 130


чт, 14 мая 2020 г. в 15:27, Alexey Schebelev <notifications@github.com>:
